src/api/drawingWrapper.js

```diff
@@ -29,8 +29,12 @@ function toParams(drawingData) {
 }
 
 export default class DrawingWapper {
-    url = 'http://localhost:3000/drawings';
-    constructor() {}
+    /* url = 'http://localhost:3000/drawings';
+    constructor() {} */
+    
+    constructor(url) {
+        this.url = url;
+    }
```

Comment:
```suggestion
    constructor() {
        this.url = 'http://localhost:3000/drawings';
    }
```
urlは定数なので引数として受け取る必要はないです！👍